libs/templates/src/lib/home/index.tsx

```diff
@@ -19,6 +20,7 @@ export function HomeTemplate() {
       <ProvenExcellenceSection />
       <CardSliderSection />
       <StaticNumberBlock />
```

Comment:
Let's refactor this to 
```suggestion
      <StaticNumberSection />
```